.travis.yml

```diff
@@ -2,9 +2,21 @@ language: python
 sudo: required
 dist: xenial
 python: 3.7.0
+services:
+- docker
+addons:
+  apt:
+    packages:
+      - docker-ce
+before_install:
+- docker swarm init
+- cp ./.env.example ./.env
```

Comment:
This shouldn't be necessary. Can we add the environment variables to Travis CI?

Reply:
Sure we can do it that way

Reply:
Updated and committed. Added the variables in Travis and everything works fine.